web/src/main/java/org/cbioportal/web/util/StudyViewFilterApplier.java

```diff
@@ -122,6 +127,11 @@ public SampleIdentifier apply(Sample sample) {
             sampleIdentifiers = filterFusionGenes(fusionGenes, sampleIdentifiers);
         }
 
+        List<StructuralVariantGeneFilter> svGenes = studyViewFilter.getSVGenes();
```

Comment:
Similar comment as before - should we combine fusionGenes and svGenes?

Reply:
agree. We do, as fusions are just one type of structure variants.